social/event/v0.1.0/schema.json

```diff
@@ -0,0 +1,138 @@
+                    "type": "string",
+                    "enum": ["confirmed", "tentative", "cancelled"]
+                },
+                "description": {
+                    "title": "Description",
+                    "description": "Detailed description of the event",
+                    "type": "string"
+                },
+                "location": {
+                    "title": "Location",
+                    "description": "Location of the event",
+                    "type": "string"
+                },
+                "creator": {
+                    "title": "Creator",
+                    "description": "Details of the event creator",
+                    "$ref": "https://common.schemas.example.com/profile/person/v0.1.0/schema.json"
+                },
+                "organizer": {
+                    "title": "Organizer",
+                    "description": "Details of the event organizer",
+                    "$ref": "https://common.schemas.example.com/profile/person/v0.1.0/schema.json"
+                },
+                "start": {
+                    "title": "Start",
+                    "description": "Start time of the event",
+                    "type": "object",
+                    "properties": {
+                        "dateTime": {
+                            "type": "string",
+                            "format": "date-time"
+                        },
+                        "date": {
+                            "type": "string",
+                            "format": "date"
+                        },
+                        "timeZone": {
+                            "type": "string"
+                        }
+                    }
+                },
+                "end": {
+                    "title": "End",
+                    "description": "End time of the event",
+                    "type": "object",
+                    "properties": {
+                        "dateTime": {
+                            "type": "string",
+                            "format": "date-time"
+                        },
+                        "date": {
+                            "type": "string",
+                            "format": "date"
+                        },
+                        "timeZone": {
+                            "type": "string"
+                        }
+                    }
+                },
+                "attendees": {
+                    "title": "Attendees",
+                    "description": "List of attendees for the event",
+                    "type": "array",
+                    "items": {
+                        "$ref": "https://common.schemas.example.com/person/v0.1.0/schema.json"
+                    }
+                },
+                "conferenceData": {
+                    "title": "Conference Data",
+                    "description": "Details about the conference associated with the event",
```

Comment:
> Include an example; I have no idea what the object data should be for this.

I don't have any idea for this as well. 
Inspired from `Google Calendar Event ConferenceData`, but not sure how to populate it for other providers.

Google calendar event conference data format: 
```
 "conferenceData": {
    "createRequest": {
      "requestId": string,
      "conferenceSolutionKey": {
        "type": string
      },
      "status": {
        "statusCode": string
      }
    },
    "entryPoints": [
      {
        "entryPointType": string,
        "uri": string,
        "label": string,
        "pin": string,
        "accessCode": string,
        "meetingCode": string,
        "passcode": string,
        "password": string
      }
    ],
    "conferenceSolution": {
      "key": {
        "type": string
      },
      "name": string,
      "iconUri": string
    },
    "conferenceId": string,
    "signature": string,
    "notes": string,
  }  
 ```
 
 I think it can be everything like `sourceData`, so assigned the `object` type.

Reply:
Ok, maybe it will become clearer when you start fetching data from the API. If so we can update the schema description later.